### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mhamidjamil**
Serial data is now in scroll able window, user can see previous data by scrolling

### CODE REVIEW

File: serial_communication/web/app.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 app = Flask(__name__)
 # Replace with your actual serial port
-ser = serial.Serial('/dev/ttyUSB0', 9600)
+ser = serial.Serial('/dev/ttyACM0', 115200)
```
Comment (wasey-rao) [inline serial_communication/web/app.py:6]: try to play with environment variables, like i want to say that create environment variables and assign them values like this:
TTGO_PORT: /dev/ttyACM0

File: temperature_work/temp.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import re
```
Comment (wasey-rao) [inline temperature_work/temp.py:1]: is it work?

File: serial_communication/web/app.py (hunk 1 above)
Comment (mhamidjamil) [reply serial_communication/web/app.py:6]: got it, I was thinking about it i might create a task for it.

**Review by wasey-rao — CHANGES_REQUESTED** [review]
if it work attach screenshot and ping me

File: temperature_work/temp.py (hunk 2 above)
Comment (wasey-rao) [reply temperature_work/temp.py:1]: No, I'm trying to resolve this feature too.

**Comment by mhamidjamil** [comment]
here is the output image
it is the output of [this code](https://github.com/mhamidjamil/orangePi/blob/main/serial_communication/templates/index.html).

File: temperature_work/temp.py (hunk 2 above)
Comment (mhamidjamil) [reply temperature_work/temp.py:1]: not yet, working on this PR is only for time related task #6

**Review by wasey-rao — APPROVED** [review]

**Pull request merged**